WordPress/WordPressTest/Services/PostAutoUploadInteractorTests.swift

```diff
@@ -119,6 +119,12 @@ class PostCoordinatorUploadActionUseCaseTests: XCTestCase {
 
         expect(action).to(equal(.upload))
     }
+
+    func testPageNotAutoUploaded() {
+        let page = createPage(.draft)
```

Comment:
Can we give a line break between each step to make more clear the arrange/act/assert steps (like the tests above)?